buildscripts/kokoro/unix.sh

```diff
@@ -90,6 +90,10 @@ if [[ -z "${ALL_ARTIFACTS:-}" ]]; then
     GRADLE_FLAGS+=" -x grpc-compiler:generateTestProto -x grpc-compiler:generateTestLiteProto"
     GRADLE_FLAGS+=" -x grpc-compiler:testGolden -x grpc-compiler:testLiteGolden"
     GRADLE_FLAGS+=" -x grpc-compiler:testDeprecatedGolden -x grpc-compiler:testDeprecatedLiteGolden"
+  elif [[ $ARCH == "ppcle_64" ]]; then
```

Comment:
Let's not duplicate the lines within. They are all the same with aarch_64. Instead, combine this if with the one above and check for aarch_64 or (`||`) ppcle_64.